spec/spec.md

```diff
@@ -560,14 +560,15 @@ _Input Descriptor Object_ also come from the same container.
 
 ### Embed Targets
 
-The following section details where the _Presentation Submission_ is to be
+The following section details where an embedded _Presentation Submission_
+, if present, ****SHOULD**** be
```

Comment:
for known target formats, ****MUST**** be

Reply:
It cannot be MUST, see comment above.